src/main/kotlin/thatline/localup/auth/service/AuthService.kt

```
@@ -38,16 +45,69 @@ class AuthService(
+                loginMethod = loginMethod,
+                ipAddress = ipAddress,
+                userAgent = userAgent,
+                deviceType = deviceType,
+                loginSuccess = success
+            )
+            
+            loginHistoryRepository.save(history)
+            
+            // 각 사용자당 최근 10개 기록만 유지
+            cleanupOldLoginHistory(email)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)